graphql-jpa-query-schema/src/main/java/com/introproventures/graphql/jpa/query/schema/impl/JpaPredicateBuilder.java

```diff
@@ -262,6 +265,19 @@ protected Predicate getDatePredicate(Path<? extends Date> root, PredicateFilter
         return null;
     }
 
+    private Predicate getUuidPredicate(Path<? extends UUID> field, PredicateFilter filter) {
+        if(filter.getValue() == null || !(filter.getValue() instanceof UUID)) {
```

Comment:
Also I think that we should not check value type in case of we already check it in `type.equals(UUID.class)`

Reply:
While that may be true, I oriented myself on the style of similar functions, i.e. the `Date` Predicate creator. If you don't mind, I would leave it like this for consistency and defensive programming reasons.

Reply:
it is ok with me